DKNightVersion/ColorTable/DKColorTable.m

```diff
@@ -18,7 +18,11 @@ @interface DKColorTable ()
 @implementation DKColorTable
 
 UIColor *DKColorFromRGB(NSUInteger hex) {
-    return [UIColor colorWithRed:((float)((hex & 0xFF0000) >> 16))/255.0 green:((float)((hex & 0xFF00) >> 8))/255.0 blue:((float)(hex & 0xFF))/255.0 alpha:1.0];
+	return [UIColor colorWithRed:((CGFloat)((hex >> 16) & 0xFF)/255.0) green:((CGFloat)((hex >> 8) & 0xFF)/255.0) blue:((CGFloat)(hex & 0xFF)/255.0) alpha:1.0];
```

Comment:
I think the indent should be 4 spaces instead of six, you can see this in this project